app/services/notify/base_send_email_service.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Notify
+  class BaseSendEmailService < ::WasteCarriersEngine::BaseService
+    include WasteCarriersEngine::ApplicationHelper
+    include ActionView::Helpers::NumberHelper
+
+    def run(registration:)
+      @registration = registration
+
+      client = Notifications::Client.new(WasteCarriersEngine.configuration.notify_api_key)
+
+      client.send_email(notify_options)
+    end
+
+    private
+
+    def validate_contact_email(registration)
```

Comment:
As this method is validating AD, it could be best kept within the back-office. If the behaviour is needed by other classes, we could extract to a mix in or service object. For now, though, am thinking it could stay here. Happy to move it elsewhere, if that would be best.

Reply:
Back office sounds fine to me 👍

Reply:
Thanks!